docs/docsite/rst/playbook_guide/playbooks_variables.rst

```diff
@@ -97,6 +97,19 @@ You will see: ``ERROR! Syntax Error while loading YAML.`` If you add quotes, Ans
 
 .. _list_variables:
 
+Boolean variables
+==============
+
+Ansible accepts a broad range of values for boolean variables: ``true/false``, ``1/0``, ``yes/no``, ``True/False``, and so on.
+While documentation examples focus on ``true/false`` to be compatible with ``ansible-lint`` default settings, you can use any of the following:
+
+ ====================== ====================================================================
+    Boolean                         Value
+ ====================== ====================================================================
+    BOOLEANS_TRUE          ('y', 'yes', 'on', '1', 'true', 't', 1, 1.0, True)
```

Comment:
Are the parentheses necessary? And occasional quotations?

Reply:
Looking at the other tables, I'd say that a lot of double backticks are missing here.

Reply:
You are suggesting I add the double backticks?

Reply:
Yes, follow the example above in the document.